src/Foundation/Http/Middleware/CheckForTrustedHost.php

```diff
@@ -0,0 +1,82 @@
+<?php namespace October\Rain\Foundation\Http\Middleware;
+
+use Config;
+use Illuminate\Http\Middleware\TrustHosts as BaseMiddleware;
```

Comment:
If we want to backport this to 5.5 I don't think this middleware exists in 5.5.

Reply:
@LukeTowers I'm happy to leave this as a L6 branch only change - I feel it's a new feature moreso than a security fix.

Reply:
@LukeTowers once this has been merged in the L6 branch, I'll do a separate PR for the 5.5 branch.

Reply:
If we have to have a separate middleware anyways we might as well just do that at this point so that we can have a clean merge back into 5.5

Reply:
Done :)